packages/manager/apps/pci-users/src/constants.ts

```diff
@@ -1,2 +1,17 @@
 export const ACTIVE_STATUS = ['ok', 'READY'];
 export const PENDING_STATUS = ['PENDING', 'CREATING', 'UPDATING', 'DELETING'];
+
+export const HORIZON_LINK: Record<string, string> = {
```

Comment:
do you need to type your constant "Record<string, string>" explicitely ? since you declare the constant with a value, typescript may infer the corresponding type by default